bin/build-lambda-dist.bash

```diff
@@ -33,8 +34,10 @@ make_zip_base() {
     (
         cd "${site_packages}"
         zip -r "${dest}" .
-        # we don't need or want pip cluttering up our lambda zip
-        zip --delete "${dest}" 'pip/*'
+        # we don't need or want pip cluttering up our lambda zip (if it exists)
+        if zip -sf "${dest}" | grep -q "pip/"; then
+            zip --delete "${dest}" 'pip/*'
+        fi
```

Comment:
Do you need a conditional check on this? Does it error if it doesn't delete anything? Would it be better just to swallow the error, you know, ask for forgiveness instead of permission? If so, you could:

```suggestion
        # we don't need or want pip cluttering up our lambda zip (if it exists)
        zip --delete "${dest}" 'pip/*' ||:
```